api/test/wfl/integration/modules/covid_test.clj

```diff
@@ -127,48 +128,88 @@
-  (-> (workloads/covid-workload-request {} {} {})
+  (-> (workloads/covid-workload-request {:dataset testing-dataset
+                                         :table   testing-table-name
+                                         :column  testing-column-name}
+                                        {:workspace            testing-workspace
+                                         :methodConfiguration testing-method-configuration}
+                                        {:workspace testing-workspace})
       (assoc :creator @workloads/email)))
 
 (deftest test-create-covid-workload-with-misnamed-source
-  (is (thrown? RuntimeException (-> (make-covid-workload-request)
-                                    (assoc-in [:source :name] "Bad_Name")
-                                    workloads/create-workload!))))
+  (is (thrown-with-msg? RuntimeException #"Failed to create workload - unknown source" (-> (make-covid-workload-request)
+                                                                                           (assoc-in [:source :name] "Bad_Name")
+                                                                                           workloads/create-workload!))))
 
 (deftest test-create-covid-workload-without-source-name
-  (is (thrown? RuntimeException (-> (make-covid-workload-request)
-                                    (assoc-in [:source :name] nil)
-                                    workloads/create-workload!))))
+  (is (thrown-with-msg? RuntimeException #"Failed to create workload - unknown source" (-> (make-covid-workload-request)
+                                                                                           (assoc-in [:source :name] nil)
+                                                                                           workloads/create-workload!))))
 
 (deftest test-create-covid-workload-without-dataset
-  (is (thrown? RuntimeException (-> (make-covid-workload-request)
-                                    (assoc-in [:source :dataset] nil)
-                                    workloads/create-workload!))))
+  (is (thrown-with-msg? RuntimeException #"Dataset is Nil" (-> (make-covid-workload-request)
+                                                               (assoc-in [:source :dataset] nil)
+                                                               workloads/create-workload!))))
+
+(deftest test-create-covid-workload-without-dataset-and-skipping-validation
+  (workloads/create-workload!
+   (workloads/covid-workload-request {:dataset nil
+                                      :table   testing-table-name
+                                      :column  testing-column-name
+                                      :skipValidation true}
+                                     {:workspace            testing-workspace
+                                      :methodConfiguration testing-method-configuration}
+                                     {:workspace testing-workspace})))
```

Comment:
This test is missing an `is` and thus failing

Reply:
Done.